src/themes/default/components/theme/blocks/Inspirations/InspirationTile.vue

```diff
@@ -1,36 +1,48 @@
 <template>
     <div class="inspiration-tile">
-        <router-link :to="{ name: 'product', params: { id: product.id, slug: product.slug }}">
-        <div class="product-image bg-lightgray">
-            <img v-bind:src="thumbnail" class="product-thumbnail"/>
-        </div>
-        <p class="mb0">{{ product.name }}</p>
-        {{ product.price | price }}
-        </router-link>
-        
+        <a :href="inspirationBlock.account_link">
```

Comment:
I think this component should be reusable and not related with instagram feed block. It's basically an image with overlay which can be used in many ways.

Reply:
Next on my ASAP list :)